src/chrome_binary.py

```diff
@@ -82,7 +82,13 @@ def download(url, name, base="."):
         revision = str(revision)
         binary_dir_path = os.path.join(path, revision)
         if os.path.exists(binary_dir_path):
-            return True
+            brp = self.get_browser_path(path, revision)
```

Comment:
Is this needed? I thought this was moved into the other pull request?
(I might be reading the diff wrong)

Reply:
I forgot to remove it. I removed it.